tests/scenarios/text-widget.spec.js

```diff
@@ -0,0 +1,59 @@
+const _ = require('lodash');
+const server = require('../server');
+const steps = require('../steps');
+
+let contentMainBlockSelector;
+let richTextSelector;
+
+module.exports = Object.assign(
+  {
+    before: (client, done) => {
+      client.resizeWindow(1200, 800);
+
+      this._server = server.create();
+      this._server.start(done);
+
+      contentMainBlockSelector = '.demo-main';
+      richTextSelector = `${contentMainBlockSelector} [data-rich-text]`;
+    },
+
+    after: (client, done) => {
+      client.end();
+      this._server.stop(done);
+    },
+  },
+  steps.main(),
+  steps.login(),
+  steps.switchLocale('en'),
+  steps.switchToDraftMode(),
+  steps.makeSubPage('Regression test'),
+  {
+    'add Rich Text widget': function(client) {
+      const addContentBtnSelector = `${contentMainBlockSelector} [data-apos-add-content]`;
+      const richTextBtnSelector = `${contentMainBlockSelector} [data-apos-add-item=apostrophe-rich-text]`;
+
+      client.click(addContentBtnSelector);
+      client.click(addContentBtnSelector);
+      client.click(richTextBtnSelector);
+      client.pause(5000);
```

Comment:
Is it necessary? Can we wait for some element?

Reply:
I've tried.
We must wait but I can't use `waitForElementVisible` because it's very hard to find out a necessary element identification ((

Reply:
I don't want to spend time on it right now